go.mod

```diff
@@ -12,4 +12,7 @@ require (
 	github.com/hashicorp/go-cty v1.4.1-0.20200414143053-d3edf31b6320
 	github.com/hashicorp/terraform-plugin-docs v0.5.1
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.10.1
+	github.com/tidwall/sjson v1.2.4
 )
+
+replace github.com/grafana/grafana-api-golang-client v0.2.6 => github.com/justinTM/grafana-api-golang-client v0.2.2-0.20220203033248-b89448800226
```

Comment:
Is this actually required? I've just checked your fork the `grafana-api-golang-client` and GitHub says it's actually behind the official one.

Reply:
https://github.com/grafana/grafana-api-golang-client/pull/55

this PR is because i opened a PR to fix the API docs to fix a 404 error with library elements by name returning array instead of single object. I will resolve Julien's review comments there (forgot about them!) and then tackle these here.

Reply:
I see 🤔 

Let's focus on that one first and then I'll come back to this one.